kube.md

```diff
@@ -25,11 +25,21 @@ Version 2 of the app introduces a payment service. The service isn't directly ac
 
 ## Deploy the Demo App
 
-You can use `kubectl` to manage Kubernetes on Docker EE, as well as using UCP. Start by cloning the GitHub repo for the demo app - connect to the **manager1** node, clone the repo and navigate to the `kubernetes` directory:
+You can use `kubectl` to manage Kubernetes on Docker EE, as well as using UCP. Start by cloning the GitHub repo for the demo app - connect to the **manager1** node, and setup kubectl to use the client bundle to authenticate with UCP.
 
 ```
-git clone https://github.com/sixeyed/docker-networking-workshop.git
+password='<password credential>'
+controller='<ucp hostname>'
+AUTHTOKEN=$(curl -sk -d '{"username":"admin","password":"'$password'"}' https://$controller/auth/login | jq -r .auth_token)
```

Comment:
If you put a space before `password=`, `controller=`, and `AUTHTOKEN=` then those commands won't be saved into the shell's history. Which is nice to avoid keeping leaving a trail.

Reply:
thanks added the spaces!